rules/thrift_0_13_0.dep

```diff
@@ -0,0 +1,10 @@
+
+SPATH       := $($(LIBTHRIFT_0_13_0)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/thrift_0_13_0.mk rules/thrift_0_13_0.dep
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+DEP_FILES   += $(shell git ls-files $(SPATH))
+
+$(LIBTHRIFT_0_13_0)_CACHE_MODE  := GIT_CONTENT_SHA
+$(LIBTHRIFT_0_13_0)_DEP_FLAGS   := $(SONIC_COMMON_FLAGS_LIST)
+$(LIBTHRIFT_0_13_0)_DEP_FILES   := $(DEP_FILES)
+
```

Comment:
Remove extra empty lines at top and bottom

Reply:
done